README.md

```diff
@@ -0,0 +1,171 @@
+pingdompy
+=========
+
+
+Python library for interacting with Pingdom services (REST API and maintenance windows).
+
+
+Features
+--------
+
+
+* Check management: create, delete, update, list
+* Maintenance windows: create, delete, list
+* Fetching outage summaries
+
+Requirements
+------------
+
+
+* Pingdom account
+* requests (0.10.8 or newer)
+
+
+Installation
+------------
+
+```
+    pip install pingdompy

```

Comment:
This won't work unless we upload pingdompy to pypi.org. Right now I think we need to suggest an install from code by clonng this repo and installing using "pip -e /path/to/cloned/repo/pingdompy"